cypress/e2e/Transcript/Access/BoardAndDoBIsRequiredToSomeBoards.feature

```diff
@@ -0,0 +1,16 @@
+Feature: The user is required to complete their profile to see the transcripts
+  Background:
+    Given I am a registered user
+    And I have selected that I have a medical board
+    And I did not provide the required <MedicalBoardInformation>
+
+  Scenario: user has empty board or date of birth details
+    Given an user did not fill the board details required by its board selection
+    When the user try to access the transcripts
+    Then a modal shows requiring to the user fill the missing board details
+    And the modal is blocking the transcript page behind
+
+  Scenarios: user is filling board or date of birth details
```

Comment:
We should mention what happens with the transcript access (I know is in the feature)

Reply:
which line you mean?

Reply:
Line 7, the scenario title.